voctocore/lib/videomix.py

```diff
@@ -341,6 +382,15 @@ def applyMixerState(self):
             mixerpad.set_property('alpha', state.alpha)
             mixerpad.set_property('zorder', state.zorder)
 
+            cropper = self.mixingPipeline.get_by_name("video_%u_cropper" % idx)
```

Comment:
In my testing there is a visible delay between the `compositor` changing config, and the `videocrop` changing config.
Moving this up below `mixerpad = …` fixes this for me. Apparently getting an element is rather slow.